MakingCoffee/src/MakingCoffeeEx1.java

```diff
@@ -0,0 +1,31 @@
+
+import java.util.Scanner;
+public class MakingCoffeeEx1 {
+
+           public static void main(String[] args) {
+            int waterPerCup = 200;
+            int milkPerCup = 50;
+            int coffeeBeansPerCup = 15;
+
+            System.out.println("Write how many ml of water the coffee machine has: ");
+            Scanner value1 = new Scanner (System.in);
```

Comment:
1. Astghik jan, there is no need to create Scanner variable each time, it is enough to have only one Scanner value declared in the top and after each println operation you can write
```
int waterValue = value.nextInt();
```
int milkValue = value.nextInt();

```
ets ...

2. After `System.out.println("Message: ")` you should have following symbol "> ", next to which you should enter the number. 
For example
```
        System.out.println("Write how many ml of water the coffee machine has: ");
        System.out.print("> ");
```